Src/WitsmlExplorer.Frontend/components/ContentViews/EditSelectedLogCurveInfo.tsx

```diff
@@ -163,79 +164,82 @@ const EditSelectedLogCurveInfo = (
   const dateTimeFormat = "yyyy-MM-dd'T'HH:mm:ss";
 
   return (
-    <Layout colors={colors}>
-      <Typography
-        style={{
-          color: `${colors.interactive.primaryResting}`
-        }}
-        variant="h3"
-      >
-        Curve Values
-      </Typography>
-      <StartEndIndex>
-        <StyledLabel label="Start Index" />
-        <StyledTextField
-          disabled={disabled}
-          id="startIndex"
-          value={startIndex}
-          // defaultValue={getDefaultValue(startIndex)}
-          variant={isValidStart ? undefined : "error"}
-          type={isTimeCurve() ? "datetime-local" : ""}
-          step="1"
-          onChange={(e: any) => {
-            onTextFieldChange(e, setStartIndex, setIsValidStart);
-          }}
-        />
-      </StartEndIndex>
-      <StartEndIndex>
-        <StyledLabel label="End Index" />
-        <StyledTextField
-          disabled={disabled}
-          id="endIndex"
-          value={endIndex}
-          // defaultValue={getDefaultValue(endIndex)}
-          type={isTimeCurve() ? "datetime-local" : ""}
-          variant={isValidEnd ? undefined : "error"}
-          step="1"
-          onChange={(e: any) => {
-            onTextFieldChange(e, setEndIndex, setIsValidEnd);
+    <EdsProvider density={"compact"}>
```

Comment:
The question is if we want some things to stay normal even when compact is selected from the settings. For example the search bar at the top left, the content of many modals, and most of the buttons. It could also make things look weird where we are using MUI instead of EDS, as all of those would not be affected by EdsProvider. 

I think the safe option is to attach the EdsProvider to the global theme setting so that you can choose compact/comfortable, but avoid changing everything else.

Reply:
Good job, this looks much better.